doc/news/_2017-10-25_0.8.20.md

```diff
@@ -52,39 +52,27 @@ Thanks again to Marvin Mall for the awesome website.
 
 #### Haskell and Ruby
 
-The Ruby binding already released earlier was greatly improved and now
-includes libtools bindings. It is the first binding going beyond Elektra's
-main API.
-
-Many [examples](https://master.libelektra.org/src/bindings/swig/ruby/examples) for
-the Ruby bindings were added.
-
-Thanks to Bernhard Denner.
-
-
-Elektra also got a new [binding](https://master.libelektra.org/src/bindings/haskell).
-Elektra also added support for [plugins](https://master.libelektra.org/src/plugins/haskell)
-to be written in Haskell. Due to generic CMake and C Code plugins can be written
-exclusively in Haskell, without any glue code.
-
-Several Haskell [examples](src/bindings/haskell/examples) already exist.
```

Comment:
i think we have to include the full link here as well otherwise the link fails:
`https://master.libelektra.org/src/bindings/haskell/examples` instead of just `src/bindings/haskell/examples`